src/formatter/types/babel__code-frame.ts

```diff
@@ -1,5 +1,5 @@
 // Base on the type definitions for @babel/code-frame 7.0
-// Project: https://github.com/babel/babel/tree/master/packages/babel-code-frame, https://babeljs.io
+// Project: https://github.com/babel/babel/tree/main/packages/babel-code-frame, https://babeljs.io
```

Comment:
although this is referencing another repo - it turns out that repo has already renamed from `master` to `main`